src/core/a-entity.js

```diff
@@ -295,7 +295,7 @@ var proto = Object.create(ANode.prototype, {
 
       ANode.prototype.load.call(this, function entityLoadCallback () {
         self.updateComponents();
-        if (self.isScene || self.parentEl.isPlaying) { self.play(); }
+        if (self.isScene || (self.parentEl && self.parentEl.isPlaying)) { self.play(); }
```

Comment:
You think it'd be possible to reproduce this in a unit test or JS consistently? Like load and unload a component, and the error previously threw every time?